### PR TITLE
Add styles

### PR DESCRIPTION
**Pull request by zsoltime**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,11 @@
+* {
```
Comment (faityworld) [inline style.css:1]: What about to align to the center?

**Review by faityworld — CHANGES_REQUESTED** [review]
I hope this button is right

**Comment by zsoltime** [comment]
Sure, forgot about that, just a moment :)

**Review by faityworld — APPROVED** [review]
good job

**Pull request merged**